### ISSUE TITLE
Fix collisions

### ISSUE THREAD

**Issue by Circl3s** [Circl3s/circl3s-super-breakout#1]

**Comment by Circl3s** [comment]
efdba882d4bed3ef76ac866bd2d1c93fe5036422 fixed the sideway bounce, sideway bounce while the paddle is moving is still bugged

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @abranhe, @Cequallium, @marcelvdh, @zeyadsharo, @nickcj931, @therahulkanyal, @fanconic, @christophermell, @VaradAC, @vir2alexport, @NathanielHill, @QuentinHuber, @DCtheTall, @devwolf75, @andersmmg, @r820704, @gneisscookie, @Fraer, @RubberFetish, @BreankingBad, @fowled, @0xc41qu3, @mmerrell, @Jnskerry, @dooglio, @dfwdamon, @gimweird, @Abgael, @peranchor, @vaipatel, @GraysonFreking, @awenblue, @sasuta, @ferch5003, @Tenyson05, @GNewRock, @AlbertWin98, @Dashulityy, @luke7oaks, @lucasarsartori, @Akshay-Rana-Gujjar, @JiaqiXiang-nb, @sabzstudio, @phanimitravadlamani, @waterVenice7, @vinicaval, @ho-lapang, @olirep6, @voz, @AntonioPoint

**Comment by fowled** [comment]
Why on earth did I receive a notification on this issue

**Comment by dooglio** [comment]
Yeah, something is totally broken at Github. I've never heard of this project before...

**Comment by andersmmg** [comment]
The description appears to have been edited, there were many random users mentioned along with an ad for what looks to be a crypto related scam.

**Comment by Circl3s** [comment]
I thought it was just a funny Github bug, but now y'all are making me a worry

**Comment by dooglio** [comment]
@Circl3s you should consider rotating your password, eh? And make sure you have 2FA turned on...

**Comment by Circl3s** [comment]
I have 2FA on both Github and my email, and both phone and email notifications, there's no way anyone could log in to my Github, let alone without me knowing. 🤔
And the description doesn't appear edited to me...

**Comment by andersmmg** [comment]
I think somehow there was a comment that is now deleted, and doesn't show on the issue. I still have a copy of the scam message in my email if you want a copy.

**Comment by dooglio** [comment]
Wow that is really strange. I would think GitHub support should be brought in on this.
